encrypted-media-respec.html

```diff
@@ -2729,6 +2732,16 @@
           <p class="note">The above step provides flexibility for user agent implementations to perform decryption at any time after an encrypted block is encountered before it is needed for playback.</p>
```

Comment:
We might want to clarify in the bullet above that we mean "encountered in the playback order" and that we're talking about a contiguous sequence of encrypted / unencrypted blocks. The corner case is that I have a sequence of decryptable / unencrypted blocks, followed by a missing block, followed by a non-decryptable block. We should wait for the missing block in the usual way (like a network stall) and only consider these algorithms with respect to the contiguous sequence when the missing block arrives and we can determine whether it is decryptable, unencrypted or undecryptable.

Maybe it is sufficient to say "This algorithm assumes blocks are encountered contiguously in playback order."